extensions/vscode/.vscodeignore

```diff
@@ -0,0 +1,16 @@
+.vscode/**
```

Comment:
Kind of feels like this should work with the opposite sense (here is what to include in the extension) but it is what it is I guess.

Reply:
Oh I fully agree. I had to fight so many accidental inclusions